ReactApp/.storybook/main.js

```diff
@@ -0,0 +1,96 @@
+  '../React-Automation-Studio/docs/widget/widget.mdx',
+  '../React-Automation-Studio/components/SystemComponents/Widgets/Widget.stories.js',
+  '../React-Automation-Studio/components/SystemComponents/PV.mdx',
+  '../React-Automation-Studio/components/SystemComponents/EpicsPV.mdx',
+  '../React-Automation-Studio/components/SystemComponents/LocalPV.mdx',
+  '../React-Automation-Studio/components/SystemComponents/Login.mdx',
+  '../React-Automation-Studio/components/SystemComponents/database/MongoDB/MongoDb.mdx',
+  '../React-Automation-Studio/components/SystemComponents/database/MongoDB/useMongoDbDeleteOne.mdx',
+  '../React-Automation-Studio/components/SystemComponents/database/MongoDB/useMongoDbInsertOne.mdx',
+  '../React-Automation-Studio/components/SystemComponents/database/MongoDB/useMongoDbUpdateOne.mdx',
+  '../React-Automation-Studio/components/SystemComponents/database/MongoDB/useMongoDbWatch.mdx',
+  '../React-Automation-Studio/docs/beamlineComponents/BeamlineComponents.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmHandlerTop.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/setup.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/environment.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/emailNotification.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/signalNotification.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/seedData.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/configJson.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/pvListJson.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/Users.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmRoleConfig.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmAdmin.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmUser.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmObserver.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/userGuide.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/implementConventions.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/serverConventions.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/serverLogging.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmNotification.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmSetupMDX.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmAreas.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmTable.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmLog.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/userNotificationMDX.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/alarmUserTable.mdx',
+  '../React-Automation-Studio/docs/alarmHandler/filterPvList.mdx',
+  '../React-Automation-Studio/components/LoadSaveComponent/LoadSave.mdx',
+  '../React-Automation-Studio/components/ArchiverDataViewer/ArchiverDataViewer.mdx',
+  // '../React-Automation-Studio/components/AlarmHandler/AlarmHandler.stories.tsx',
```

Comment:
Do we need to keep those commented-out lines? If so, would it be possible to add explanation what it means?